src/lib/tools/path.js

```diff
@@ -12,7 +12,10 @@ function createNestedObject(object, keys, defaultValue) {
 }
 
 function applyJsonPath(object, path = '$') {
-  return jp.value(object, path) || {};
+  if (object) {
```

Comment:
I feel we should check if `typeof obj === 'object'` instead, is that ok for you? (otherwise an empty string would still go there)